java/maven/src/org/netbeans/modules/maven/api/PluginPropertyUtils.java

```diff
@@ -652,21 +657,48 @@ public List<Dependency> build(Xpp3Dom configRoot, ExpressionEvaluator eval) {
                     item.setGroupId(g.getValue());
                     item.setLocation(PROP_GROUP_ID, (InputLocation)g.getInputLocation());
                 }
+
+                Dependency dependencyFromDependencyManagement = null;
+                if(dependencyManagement != null && dependencyManagement.getDependencies() != null) {
+                    dependencyFromDependencyManagement =
+                            dependencyManagement
+                            .getDependencies()
+                            .stream()
+                            .filter(d -> {
+                                return Objects.equals(item.getGroupId(), d.getGroupId())
+                                        && Objects.equals(item.getArtifactId(), d.getArtifactId());
+                            })
```

Comment:
nitpick: expression form would produce less brackets
```java
                            .filter(d -> Objects.equals(item.getGroupId(), d.getGroupId())
                                      && Objects.equals(item.getArtifactId(), d.getArtifactId()))
```

Reply:
Yeah and as I pushed an update I tried that variant, but it has a major drawback: Automatic formatting without braces looks ugly, with brackets it is good. Given how often I reformat, I prefer the stable format at at price of a single line and some brackets.